iron/distribution.yaml

```diff
@@ -2809,6 +2809,21 @@ repositories:
       url: https://github.com/DFKI-NI/mir_robot.git
       version: rolling
     status: developed
+  mocap_optitrack:
+    doc:
+      type: git
+      url: https://github.com/ros-drivers/mocap_optitrack.git
+      version: foxy-devel
+    release:
+      tags:
+        release: release/iron/{package}/{version}
+      url: https://github.com/ros-drivers-gbp/mocap_optitrack-release.git
+      version: 1.0.0-2
+    source:
+      type: git
+      url: https://github.com/ros-drivers/mocap_optitrack.git
+      version: foxy-devel
+    status: unmaintained
```

Comment:
Release a new package (or new-to-distro) and marking it `unmaintained` seems strange.  Any reason here?

Reply:
Indeed it does.  I no longer have access to the hardware to test on.  The software builds but I can't test the functionality.  I was going to start looking for a new maintainer that has access to the hardware.